assets/css/style.css

```diff
@@ -338,6 +376,18 @@ body {
     text-decoration: none;
 }
 
+@media(device-width: 768px) and (device-height: 1024px) {
+    ::-webkit-scrollbar {
```

Comment:
Actually, this applies to **all** scrollbars on the page. We only want to affect the language dropdown one!